### PR TITLE
Create eurosurveillance.csl

### PR DESCRIPTION
**Pull request by Polperobis**
This is a new style

### CODE REVIEW

File: eurosurveillance.csl [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    <title>Eurosurveillance</title>
+    <id>http://www.zotero.org/styles/eurosurveillance</id>
+    <link href="http://www.zotero.org/styles/eurosurveillance" rel="self"/>
+    <link href="http://www.zotero.org/styles/springer-vancouver-brackets" rel="template"/>
+    <link href="https://www.eurosurveillance.org/for-authors#References" rel="documentation"/>
+    <author>
+      <name>Vincent Auvigne</name>
+      <email>vincent.auvigne@ekipaj.com</email>
+    </author>
+    <contributor>
```
Comment (POBrien333) [inline eurosurveillance.csl:13]: You can delete Charles from the contributors. 
Contributors are acknowledged through the template link in line 7.

**Comment by Polperobis** [comment]
I do not understand why the tests failed. Can someone help me ?

**Comment by POBrien333** [comment]
`expected et-al-min (7) and et-al-use-first () to be of same type`

Basically, if you have et-al-min, you also need et-al-use-first.
You have that correctly set on `<bibliography....`, but have a look at your line 2. I would assume you'd just want to delete that there, since it's handled further down anyway.

File: eurosurveillance.csl [hunk 2]
```diff
@@ -0,0 +1,265 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en">
```
Comment (POBrien333) [inline eurosurveillance.csl:2]: I think it requires a specific English language.
en-GB or en-US are the most common ones.
Note that they can change your style, especially how dates are shown.

Comment (Polperobis) [reply eurosurveillance.csl:2]: I do not understand your comment, as I had later on a message "All checks have passed" without changing the default-locale

**Comment by adam3smith** [comment]
Thanks! Given that this explicitly points to the NLM guidelines that the Vancouver styles follow, is there really none of the existing Vancouver styles we could link here? Seems unnecessary to create yet another slightly modified version if the journal links to the exact same examples.

**Comment by adam3smith** [comment]
@Polperobis see my question above

**Comment by adam3smith** [comment]
Closing for inactivity; we're happy to re-open if you come back to this and reply to the questions above -- just post here.

**Pull request closed without merging**